lib/core/connect.js

```diff
@@ -99,6 +99,8 @@ function buildConnector ({ maxCachedSessions, socketPath, timeout, ...opts }) {
         servername,
         session,
         localAddress,
+        // TODO(HTTP/2): Should we support h2c?
```

Comment:
Is this something that we should explore or focus only on `h2`?

Reply:
this can be handled in a later pr